README.md

```
@@ -25,6 +25,12 @@ CruiseKube is an intelligent Kubernetes resource optimization controller that au
 >
 > https://github.com/truefoundry/CruiseKube/issues/new?template=adopter.yaml
 
+
+## ▶️ Live Demo
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)